pkg/projects/resource_project.go

```diff
@@ -28,7 +30,7 @@ type Project struct {
 	DisplayName            string           `hcl:"display_name" json:"display_name"`
 	Description            string           `hcl:"description" json:"description"`
 	AdminPrivileges        *AdminPrivileges `hcl:"admin_privileges" json:"admin_privileges"`
-	StorageQuota           int              `hcl:"max_storage_in_gigabytes" json:"storage_quota_bytes"`
+	StorageQuota           int              `hcl:"max_storage_in_gibabytes" json:"storage_quota_bytes"`
```

Comment:
This only matters if you're going to use the universal packer that's in RT - which, that code no longer needs this. It snake cases for you